dotcom-rendering/src/components/ContainerTitle.tsx

```diff
@@ -23,9 +23,9 @@ type Props = {
 	editionId?: EditionId;
 };
 
-const linkStyles = css`
+const linkStyles = (fontColour?: string) => css`
 	text-decoration: none;
-	color: ${neutral[7]};
+	color: ${fontColour ?? neutral[7]};
```

Comment:
Would this colour not already get apllied on https://github.com/guardian/dotcom-rendering/pull/7891/files#diff-214b94c0ffdf0184ea727aa00221bbef7180a03ce71a8fce2f5f10c1f17f50eeR37

Wondering if we can just not apply a colour at all in the link styles and leave it up to the headerStyles 🤔

Reply:
Nice suggestion! I only kept 

```
const linkStyles = css`
	text-decoration: none;
`;
```

to avoid the ugly blue line under the link. Other thank that, no need to pass any CSS more there!